tests/ui/perfherder/alerts-view/status_dropdown_test.jsx

```diff
@@ -242,15 +266,18 @@ test('Triage to countdown shows 3 working days left when alert is createad on Su
   expect(dueDateStatusText).toBe('Working days left: 3');
 });
 
-test('Triage to countdown shows 3 working days left when the due date its either Saturday or Sunday and the alert is created on Tuesday', async () => {
+test('Triage to countdown shows 3 working days left when the due date its either Saturday and the alert is created on Tuesday', async () => {
```

Comment:
When an alert is created on Tuesday then the due date is on Friday, not Satuday so this is not accurate.